ddtrace/settings/config.py

```diff
@@ -312,6 +313,12 @@ def __init__(self):
             # Replaces the default otel api runtime context with DDRuntimeContext
             # https://github.com/open-telemetry/opentelemetry-python/blob/v1.16.0/opentelemetry-api/src/opentelemetry/context/__init__.py#L53
             os.environ["OTEL_PYTHON_CONTEXT"] = "ddcontextvars_context"
+            # make propagation work automatically when otel enabled
+            if _PROPAGATION_STYLE_W3C_TRACECONTEXT not in self._propagation_style_extract:
+                self._propagation_style_extract.append(_PROPAGATION_STYLE_W3C_TRACECONTEXT)
```

Comment:
Order only matters if otherwise we'd be making a mistake by parsing datadog context instead of tracecontext, which I don't think is the case here. I think we should continue to prioritize `datadog` context, so that we don't run into the same issue we encountered when making the default `tracecontext,datadog`.